src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml

```diff
@@ -49,27 +52,48 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-lg-4">
                         <div class="d-flex flex-wrap align-items-center justify-content-stretch">
-                            <div id="@Html.Id("SelectEncryptionMethod")" class="form-group flex-fill" asp-validation-class-for="EncryptionMethod">
-                                <label asp-for="EncryptionMethod">@T["Encryption method to use"]</label>
-                                <select asp-for="EncryptionMethod" class="form-control" disabled="@Model.AutoSelectEncryption">
-                                    <option value="@SmtpEncryptionMethod.None" selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod ? true : false)">
-                                        @T["None"] - @T["Connect to server using insecure connection."]
-                                    </option>
-                                    <option value="@SmtpEncryptionMethod.SSLTLS" selected="@(SmtpEncryptionMethod.SSLTLS == Model.EncryptionMethod ? true : false)">
-                                        @T["SSL/TLS"] - @T["Connect to server using SSL/TSL secure connection"]
-                                    </option>
-                                    <option value="@SmtpEncryptionMethod.STARTTLS" selected="@(SmtpEncryptionMethod.STARTTLS == Model.EncryptionMethod ? true : false)">
-                                        @T["STARTTLS"] - @T["Connect to server using insecure connection and upgrade to secure using SSL/TLS"]
-                                    </option>
-                                </select>
-                                <span asp-validation-for="EncryptionMethod"></span>
-                                <span class="hint">@T["The encryption method used when connecting to mail server."]</span>
+                            <div class="form-group flex-fill" asp-validation-class-for="ProxyPort">
+                                <label asp-for="ProxyPort">@T["Proxy port number"]</label>
+                                <input asp-for="ProxyPort" class="form-control" type="number" step="any" min="0" max="65535" />
+                                <span asp-validation-for="ProxyPort"></span>
+                                <span class="hint">@T["The proxy port is optional."]</span>
                             </div>
                         </div>
                     </div>
                 </div>
+
+                <div id="@Html.Id("SelectEncryptionMethod")" class="form-group" asp-validation-class-for="EncryptionMethod">
+                    <label asp-for="EncryptionMethod">@T["Encryption method to use"]</label>
+                    <select asp-for="EncryptionMethod" class="form-control" disabled="@Model.AutoSelectEncryption">
+                        <option value="@SmtpEncryptionMethod.None" selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod ? true : false)">
```

Comment:
I think razor is clever enough to just use `selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod)"`

Reply:
Saw that it was not part of the PR

Reply:
Seb can we merge this, then I can look for the places that may need to change